docs/utils/loadable.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: loadable
+---
+
+For some reason, you may not want to profit from the React's `Suspense` core handling of jotai, and prevent your app from suspending when using async atoms.
```

Comment:
This comes off as a little negative to me ("not want to profit"), I think there are lots of valid use-cases for `loadable`. 

You could say something like

```
If you don't want async atoms to suspend or throw to an error boundary (for example, for finer-grained control of loading and error logic), you can use the `loadable` util. It returns a value with one of three states: `loading`, `hasData` and `hasError`.
```

Reply:
@Pinpickle Sounds way better!